spec/yaml/swiftnav/sbp/imu.yaml

```diff
@@ -115,3 +115,63 @@ definitions:
                       - 2: +/- 8g
                       - 3: +/- 16g
                       - 4: +/- 6g
+
+ - MSG_IMU_COMP:
+    id: 0x0905
+    short_desc: Compensated IMU data
+    desc: >
+      Data from the Inertial Measurement Unit, containing accelerometer and
+      gyroscope readings compensated for estimated errors and constant 
+      physical effects. The output is valid for the center of navigation.
```

Comment:
For being unambiguous, let's specify object frame, reference frame, and resolving frame. How about this?
'The output is valid for inertially referenced center of navigation(imu body frame) represented in vehicle body frame'

Reply:
corrected! 

If we ever need to support a transformation to VRP, we will use one of the reserved flags and adjust this description.